librubyfmt/src/file_comments.rs

```diff
@@ -180,7 +181,7 @@ impl FileComments {
                 sled.add_line(l);
             }
             _ => {
-                self.other_comments.insert(line_number, l);
+                self.other_comments.push((line_number, l));
```

Comment:
I _think_ this is the only place comments get added to this field -- WDYT about adding a debug assertion here that either this vec is empty or the line number that we're pushing is greater than the last line number in the vector?  Just as a way to make sure this is sorted, since the bits below depend on that invariant.